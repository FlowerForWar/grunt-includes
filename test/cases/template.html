<html>
  <head>
    <title>Includes test</title>
  </head>
  <body>
    <div class="header">
      include "header.html"
    </div>

    <div class="body">
      <p>Hello world</p>
    </div>

    <div class="footer">
      include "footer.html"
    </div>
  </body>
</html>
