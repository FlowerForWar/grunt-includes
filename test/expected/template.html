<html>
  <head>
    <title>Includes test</title>
  </head>
  <body>
    <div class="header">
      Template start header.html
      <header>
        <h1>Header</h1>
      </header>
      Template end
    </div>

    <div class="body">
      <p>Hello world</p>
    </div>

    <div class="footer">
      Template start footer.html
      <footer>
        <h1>Footer</h1>
      </footer>
      Template end
    </div>
  </body>
</html>
